test/e2e/arktos/multi_tenancy/test_suites/multi_tenancy_controller/test_deployment_replicaset_controller.yaml

```diff
@@ -0,0 +1,141 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Deployment & ReplicaSet Controller Tests ~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies the multi-tenancy deployment & replicaset controller. 
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  test_ns: random_8
+  test_tenant: random_8
+
+###########################################################################################################
+# test setup
```

Comment:
comment here tries to tell test steps aprt; nice attempt. Could it be part of the test case output too, hence more visible and useful when looking at test case output?

Reply:
That is good suggestion. I will add this feature to the test tool.